src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// DragAndDrop controls the drag objet and data.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class DragAndDrop : BaseHandle
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void DragAndDropEventHandler(View targetView, DragEvent dragEvent);
+        private delegate void InternalDragAndDropEventHandler(global::System.IntPtr dragEvent);
+        private Dictionary<View, InternalDragAndDropEventHandler> targetEventDictionary = new Dictionary<View, InternalDragAndDropEventHandler>();
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        private DragAndDrop() : this(Interop.DragAndDrop.New(), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        private DragAndDrop(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static DragAndDrop Instance { get; } = new DragAndDrop();
+
+        /// <summary>
+        /// Starts drag and drop.
+        /// </summary>
+        /// <param name="sourceView">The soruce view</param>
+        /// <param name="shadowView">The shadow view for drag object</param>
+        /// <param name="data">The data to send</param>
+        /// <returns>true if starting drag operation is success. otherwise, false.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void StartDragAndDrop(View sourceView, View shadowView, string data)
+        {
+            Interop.DragAndDrop.StartDragAndDrop(SwigCPtr, View.getCPtr(sourceView), View.getCPtr(shadowView), data);
```

Comment:
제가 bool로 실패 결과를 알리지 말라고 한것은 그냥 리턴값을 무시하라는 의미가아니고 native에서 전달하는 return값이 실패인 경우 exception을 throw하여 실패를 알리라는 의미였습니다.

```suggestion
            if (!Interop.DragAndDrop.StartDragAndDrop(SwigCPtr, View.getCPtr(sourceView), View.getCPtr(shadowView), data))
                        throw new InvalidOperationException("Fail to StartDragAndDrop");
```